File: isa-mrs-2022/Spring-Boot-Backend/backend/src/main/java/rs/ac/uns/ftn/backend/dto/response/CottageReservationCalendarDTO.java
package rs.ac.uns.ftn.backend.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import rs.ac.uns.ftn.backend.dto.request.SimpleActionDTO;
import rs.ac.uns.ftn.backend.model.CottagePricelist;
import rs.ac.uns.ftn.backend.model.CottageResevation;

import java.time.LocalDate;
import java.util.ArrayList;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CottageReservationCalendarDTO {

    private List<CottageResevation> cottageResevations= new ArrayList<>();

    private List<SimpleActionDTO> cottageActionsReservated= new ArrayList<>();

    private List<SimpleActionDTO> cottageActionsUnReservated= new ArrayList<>();

}

File: isa-mrs-2022/Angular-Frontend/angular-frontend/src/app/models/response/http-cottage-response/CottageReservationCalendarDTO.ts
import { SimpleActionDTO } from './../../util/SimpleActionDTO';
import { CottageResevationDTO } from "./cottage-reservation";

export class CottageReservationCalendarDTO {

    cottageResevations: CottageResevationDTO[];

    cottageActionsReservated: SimpleActionDTO[];

    cottageActionsUnReservated: SimpleActionDTO[];

}